docs/pages/access-controls/guides/impersonation.mdx

```diff
@@ -244,8 +271,8 @@ spec:
       '*': '*'
 ```
 
-This role will work with SSO users, but we can test locally, by updating Alice's account
-entry with traits:
+While user traits typically from from an external identity provider, we can test
+with local user `alice` by manually updating Alices's account with traits.
```

Comment:
with a local user?

Reply:
I'm not sure what this comment means. The guide has us manually updating spec.traits for a local user, right?

Reply:
Sorry, I was wondering if `a` article is omitted in this sentence: "we can test with **a** local user, ...